utils/testdata/interfaceTestNoName.sol

```diff
@@ -0,0 +1,4 @@
+interface  {
```

Comment:
This is not valid Solidity, we should default to `interface Interface` if no name is supplied